index.html

```diff
@@ -0,0 +1,319 @@
+              </ul>
+            </nav>        
+    </header>
+    <main>
+        <section class="works">
+         <div class="headline">
+            <div class="headline-box">
+              <h2>
+                  I’m Jorum<br/>Glad to see you!</h2>
+                  
+              <p class="details">
+                I’m a software developer! I can help you build a product , feature
+                or website Look through some of my work and experience! If you like
+                what you see and have a project you need coded, don’t hestiate to
+                contact me.
+              </p>
+              <p class="social-link-heading">LET’S CONNECT</p>
+              <div class="sociol-logo">
+                <ul class="connect-logo">
+                    <li>
+                        <a href="#"><img src="images/twitterIcon.svg" alt=""></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/linkedInIcon.png" alt=""></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/mIcon.svg" alt=""></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/gitHubIcon.svg" alt=""></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/peaceIcon.svg" alt=""></a>
+                    </li>
+                </ul>
+            </div>
+            </div>
+        </div>       
+        </section>
+        <section class="works">
+            <div class="works-detail">            
+            <h2 class="main-topic">Tonic</h2><br>
+            <div class="works-intro">
+                <h2>Canopy</h2>
+                <ul>
+                    <li>
+                        <img src="images/Counter.png" alt="Counter"></li>
+                        <li><img src="images/backend.png" alt=""></li>
+                    
+                    <li>
+                        <img src="images/Counter.png" alt="counter"></li>
+                        <li><img src="images/2015.png" alt=""></li>                    
+                </ul>
+            </div>
+                <br>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="language">
+                    <div class="lang-icon">
+                    <ul>
+                        <li>
+                            <img src="images/html1.png" alt="html">
+                        </li>
+                        <li>
+                            <img src="images/css1.png" alt="css">
+                        </li>
+                        <li>
+                            <img src="images/javascript1.png" alt="javaScript">
+                        </li>
+                    </ul>
+                </div>
+                    <button type="button"><img src="images/SeeProject.png" alt=""></button>
```

Comment:
- Please try to make your mobile design look similar to that of the Figma design.

#### Mobile view
- Please on the header, there should be a hamburger menu as in the Figma design. 
- Please try to align the elements in the header section properly as it is in the Figma design.
- Please on the `project section`, please make each project card fit the mobile screen, try setting a width on them in percentage. Also the technology tags, some are not properly designed, please try to fix them.
- Please try to align the elements properly on the `about section` as it is on the Figma design.
- Please on the `contact section` allow the section to fit properly on the screen and in the Figma design.

##### Your design
https://user-images.githubusercontent.com/58771507/173184284-47ce4de5-42a2-4cb3-a1e9-50610af7b8bf.mp4

##### Figam design
https://user-images.githubusercontent.com/58771507/173184224-70033cff-aa85-4d66-a681-8be0980e307f.mp4

Reply:
## In mobile version
- [ ] Please implement these changes as stated by previous reviewer

> Please on the header, there should be a hamburger menu as in the Figma design and in desktop remove it, you can use display: none and block for that.

> Please on the `project section`, please make each project card fit the mobile screen, try setting a width on them in percentage. Also the technology tags, some are not properly designed, please try to fix them to match the design.